portality/bll/services/journal.py

```diff
@@ -145,16 +151,20 @@ def csv(self, prune=True):
             mainStore.store(container_id, filename, source_path=out)
             url = mainStore.url(container_id, filename)
         finally:
-            tmpStore.delete_file(container_id, filename) # don't delete the container, just in case someone else is writing to it
+            tmpStore.delete_file(container_id,
+                                 filename)  # don't delete the container, just in case someone else is writing to it
 
         action_register = []
         if prune:
             def sort(filelist):
                 rx = "journalcsv__doaj_(.+?)_utf8.csv"
-                return sorted(filelist, key=lambda x: datetime.strptime(re.match(rx, x).groups(1)[0], FMT_DATETIME_SHORT), reverse=True)
+                return sorted(filelist,
+                              key=lambda x: datetime.strptime(re.match(rx, x).groups(1)[0], FMT_DATETIME_SHORT),
```

Comment:
should dates library need to be used instead datetime

Reply:
did you mean replace `datetime.strptime` to `dates.parse`? 
behavior of `datetime.strptime` and `dates.parse` is different, `dates.parse` will auto guess the format of date.
`datetime.strptime` is low level, faster and more predictable to parse the date, we can open for developer to choice which method is better in this case.